### DISCUSSION TITLE
PrimeNG 18 without SASS and backward compatibility ?

### DISCUSSION THREAD

**Discussion by djay1977** [primefaces/community#2641] [PrimeNG]
Hello,
With the new theming of PrimeNG 18 without SASS.
Will the library be backward compatible with the use of SASS?
Could we still use SASS and update PrimeNG?
Regarding the templates (Ultima and others), will these be reviewed because they are based on the SASS API?

Thanks

**Comment by JIBIN-P** [comment]
The templates will be converted by the end of 2024 as stated in the v18 docs

https://v18.primeng.org/guides/migration#breaking:~:text=Premium%20Templates,of%20the%202024.

I think you should be able to use sass as that part is handled by angular and prime-ng won't interfere with those settings.

**Comment by qwadrox** [comment]
@djay1977 [It won't be backward compatible](https://v18.primeng.org/guides/migration#breaking). theme.css won't exist anymore, you don't have to import anything. If you had a custom theme created by the SASS API you would need to reimplement the theme relying on the new API. See the customization section for that: https://v18.primeng.org/theming#customization
